ecm-wcm-extension/src/main/webapp/WEB-INF/conf/dms-extension/dms/dms-actions-configuration.xml

```diff
@@ -67,8 +67,23 @@
                     		</collection>
                    		</field>
                     </object>  
-                  </value>                                  
+                  </value>
 
+                  <value>
+                    <object type="org.exoplatform.services.cms.actions.impl.ActionConfig$Action">
+                      <field  name="type"><string>exo:move</string></field>
+                      <field  name="name"><string>moveNode</string></field>
+                      <field  name="description"><string>cut and past a node in a new destination</string></field>
+                      <field  name="srcWorkspace"><string>collaboration</string></field>
+                      <field  name="srcPath"><string>/</string></field>
+                      <field  name="isDeep"><boolean>true</boolean></field>
+                      <field  name="lifecyclePhase">
+                        <collection type="java.util.ArrayList">
+                          <value><string>node_moved</string></value>
+                        </collection>
+                      </field>
+                    </object>
+                  </value>
```

Comment:
Can you explain what this action is for ? I'm not sure to understand.
Thanks

Reply:
> Can you explain what this action is for ? I'm not sure to understand. Thanks
This action need to be registered in `ActionCatalog` : https://github.com/exoplatform/jcr/blob/develop/exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/ext/action/SessionActionInterceptor.java#L56 
declared for exo:move type and dispatched event is node_moved its name is `moveNode` which i added it in eventTypes of the `FileIndexerAction`
https://github.com/exoplatform/jcr/blob/develop/exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/ext/action/SessionActionInterceptor.java#L512  

if we don't register it the event will not be launched :
https://github.com/exoplatform/jcr/blob/develop/exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/ext/action/SessionActionInterceptor.java#L563